src/Controller/DebugKitController.php

```diff
@@ -43,7 +43,7 @@ public function beforeFilter(EventInterface $event): void
         // ignore it, only if `DebugKit.ignoreAuthorization` is set to true
         $authorizationService = $this->getRequest()->getAttribute('authorization');
         if ($authorizationService instanceof AuthorizationService) {
-            if (Configure::read('DebugKit.ignoreAuthorization')) {
+            if (Configure::read('DebugKit.ignoreAuthorization') !== false) {
```

Comment:
We had defaulted to off to take care of the inevitable situation of someone deploying with debug on. The hope was that folks wouldn't also deploy this config option.